.circleci/config.yml

```diff
@@ -231,4 +231,8 @@ workflows:
         - equal: [ scheduled_pipeline, << pipeline.trigger_source >> ]
         - equal: [ "update-native-dependencies", << pipeline.schedule.name >> ]
     jobs:
-      - automatic-dependency-update 
\ No newline at end of file
+      - automatic-dependency-update 
+
+  danger:
+    jobs:
+      - revenuecat/danger
```

Comment:
Will this also run on branches without PRs? Not sure if we have that option enabled in this repo as well...

Reply:
Good point! I just enabled it :)